Assets/MRTK/Examples/Demos/StandardShader/Scenes/MaterialGallery.unity

```diff
@@ -4582,72 +4778,267 @@ Transform:
     type: 3}
   m_PrefabInstance: {fileID: 314894433}
   m_PrefabAsset: {fileID: 0}
---- !u!1 &1104263734
-GameObject:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInstance: {fileID: 0}
-  m_PrefabAsset: {fileID: 0}
+--- !u!21 &1083148308
+Material:
   serializedVersion: 6
-  m_Component:
-  - component: {fileID: 1104263735}
-  m_Layer: 0
-  m_Name: RimLightTransparent
-  m_TagString: Untagged
-  m_Icon: {fileID: 0}
-  m_NavMeshLayer: 0
-  m_StaticEditorFlags: 0
-  m_IsActive: 1
---- !u!4 &1104263735
-Transform:
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 1104263734}
-  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: -1, y: 0, z: 3}
-  m_LocalScale: {x: 1, y: 1, z: 1}
-  m_Children:
-  - {fileID: 110133850}
-  - {fileID: 1588100095}
-  m_Father: {fileID: 1229001242}
-  m_RootOrder: 17
-  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
---- !u!1001 &1107606850
-PrefabInstance:
-  m_ObjectHideFlags: 0
-  serializedVersion: 2
-  m_Modification:
-    m_TransformParent: {fileID: 489252457}
-    m_Modifications:
-    - target: {fileID: 1188282787330574, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_Name
-      value: Placard
-      objectReference: {fileID: 0}
-    - target: {fileID: 1188282787330574, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_StaticEditorFlags
-      value: 0
-      objectReference: {fileID: 0}
-    - target: {fileID: 1815568970368836, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_StaticEditorFlags
-      value: 0
-      objectReference: {fileID: 0}
-    - target: {fileID: 1937578906433374, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_StaticEditorFlags
-      value: 0
-      objectReference: {fileID: 0}
-    - target: {fileID: 4085006294760784, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_LocalPosition.x
-      value: 0
-      objectReference: {fileID: 0}
-    - target: {fileID: 4085006294760784, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_LocalPosition.y
-      value: 1
-      objectReference: {fileID: 0}
-    - target: {fileID: 4085006294760784, guid: 28f445004b874b329a03cd0e3cc63e5d, type: 3}
-      propertyPath: m_LocalPosition.z
-      value: 0
+  m_Name: ShaderBallClippingBox (Instance)
```

Comment:
@vaoliva @Cameron-Micka Should these instanced materials be getting serialized into the scene?

Reply:
This was the old (current?) behavior of the MaterialInstance. But, @vaoliva may have addressed this with her latest changes?

Reply:
Yeah, it seems like there are some other scenes that both had and continue to have them serialized. This was the only one that looked like it had new materials serialized, so I wanted to make sure it's expected!

Reply:
@keveleigh thanks for spotting this. I will check a change I have made to solve an issue with text mesh pro.

Reply:
Chatted with @vaoliva offline: I'm gonna go ahead and merge this PR, then we can clean up any existing instances after the change referenced above

Reply:
Opened #8527 to investigate the issue.